pkg/authn/delegating.go

```diff
@@ -62,6 +55,6 @@ func NewDelegatingAuthenticator(client authenticationclient.TokenReviewInterface
 		APIAudiences:                       authenticator.Audiences(authn.Token.Audiences),
 	}
 
-	authenticator, _, err := authenticatorConfig.New()
-	return authenticator, err
+	auth, _, err := authenticatorConfig.New()
```

Comment:
eh,  if I'm reviewing, I prefer the original.  We don't charge extra for letters and authenticator and authorizer both share auth.

Reply:
this shadows the "authenticator" package import but i am agnostic as well.